docs/reference/installation.rst

```diff
@@ -297,12 +261,11 @@ Then, add a new custom firewall handlers for the admin:
             # default login area for standard users
 
             # This firewall is used to handle the public login area
```

Comment:
@jordisala1991 i think this section should be dropped? this Bundle doesn't have the Actions anymore to be used as frontend/public area

Reply:
You're right! I will remove it.

Reply:
Removed.